### ISSUE TITLE
jsx-no-jsx-as-prop doesn't trigger for react fragments

### ISSUE THREAD

**Issue by petey** [cvazac/eslint-plugin-react-perf#71]
The `jsx-no-jsx-as-prop` does not trigger on JSX inside of React fragments when the fragment shorthand is used, e.g.:

```jsx
// No error - bad
const bar = (
    <Foo
        jsx={
            <>
                <div />
                <div />
            </>
        }
    />
);
```

It does capture the explicit React.Fragment:
```jsx
// Errors - good
const bar = (
    <Foo
        jsx={
            <React.Fragment>
                <div />
                <div />
            </React.Fragment>
        }
    />
);
```
